src/jaegertracing/samplers/Config.h

```diff
@@ -47,7 +47,7 @@ class Config {
 
     static constexpr auto kJAEGER_SAMPLER_TYPE_ENV_PROP = "JAEGER_SAMPLER_TYPE";
     static constexpr auto kJAEGER_SAMPLER_PARAM_ENV_PROP = "JAEGER_SAMPLER_PARAM";
-    static constexpr auto kJAEGER_SAMPLER_MANAGER_HOST_PORT_ENV_PROP = "JAEGER_SAMPLER_MANAGER_HOST_PORT";
+    static constexpr auto kJAEGER_SAMPLING_ENDPOINT_ENV_PROP = "JAEGER_SAMPLING_ENDPOINT";
```

Comment:
We cannot simply replace the name, it will break existing deployments. The old property needs to be supported as deprecated.

Reply:
Yes we can simply replace on the cpp client, because the env props are recent and the loading of this one was not coded yet, it was unused in the code, so if anyone was using it in their software, they would have noticed that it had no effect. Hence my code and unit tests that now make it work.